LocalReferenceLinkRel/explainer.md

```diff
@@ -148,6 +145,27 @@ the shadow root where they are defined, as illustrated by the following examples
   Styles defined inside the sibling Shadow Root are not applied, so "Inside Sibling Shadow DOM" is not blue.
 </p>
 
+These scoping behaviors do not apply to [Declarative CSS Modules](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ShadowDOM/explainer.md)
+because modules always have global scope. Shadow DOM scoping could be modified though other means, as discussed
+in [this thread](https://github.com/whatwg/html/issues/11364). All of the suggestions in that thread would
```

Comment:
I wonder if we should spell these out here instead of linking to the issue, or if there is one proposal there in particular, spelling that out here more explicitly and linking to the issue for other options.

The main reason being that if that thread ends up growing in length, it would be good to have some form of summary or main takeaway here, as well.

Reply:
Good idea, I listed my preferred option there, as well as a reference to the other examples